test/Feature/BFSSearcherAndDFSSearcherInterleaved.c

```diff
@@ -0,0 +1,42 @@
+// RUN: %llvmgcc %s -emit-llvm -g -c -o %t.bc
+// RUN: rm -rf %t-bfs.klee-out
+// RUN: rm -rf %t-dfs.klee-out
+// RUN: rm -rf %t-bfs-dfs.klee-out
+// RUN: rm -rf %t-dfs-bfs.klee-out
+// RUN: %klee -libc=uclibc -output-dir=%t-bfs.klee-out -search=bfs %t.bc >%t-bfs.out
+// RUN: %klee -libc=uclibc -output-dir=%t-dfs.klee-out -search=dfs %t.bc >%t-dfs.out
+// RUN: %klee -libc=uclibc -output-dir=%t-bfs-dfs.klee-out -search=bfs -search=dfs -use-batching-search %t.bc >%t-bfs-dfs.out
+// RUN: %klee -libc=uclibc -output-dir=%t-dfs-bfs.klee-out -search=dfs -search=bfs -use-batching-search %t.bc >%t-dfs-bfs.out
```

Comment:
I'm worried this test might be fragile because it is relying on `-use-batching-search` which uses a time and instruction budget. We can hide the time budget by using a very large `-batch-time=` argument but I'm not sure about the instruction budget (defaults to `10000` instructions). This could vary between Clang versions.